src/templates/legal/regulation.html.tt

```diff
@@ -0,0 +1,198 @@
+                                        <img src="[% request.url_for('images/pages/regulation/pillar.svg') %]">
+                                    </div>
+                                    <div class="gr-9 gr-centered center-text quote-text">
+                                        <a download="[% true %]" href="[% request.url_for('/download/BIEL_Pillar_3_Report.pdf') %]" target="_blank">
+                                            [% l("Pillar 3 disclosure report") %]
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <h3 class="margin-top">[% l("Prospectus") %]</h3>
+                    <div>
+                        <p>[% l("Read about the features, characteristics, benefits, and risks of binary options in the prospectus of Binary Investments (Europe) Limited below. Included in the \"Schedules\" section are investment examples of contract types to help you get to grips with the basics of binary options.") %]</p>
+                        <div class="gr-4 gr-12-m gr-6-p gr-centered">
+                            <div class="xquote border-dark-gray">
+                                <div class="gr-row">
+                                    <div class="gr-4">
+                                        <img src="[% request.url_for('images/pages/regulation/prospectus.svg') %]">
+                                    </div>
+                                    <div class="gr-6 gr-centered center-text quote-text">
+                                        <a download="[% true %]" href="[% request.url_for('/download/BIEL_Prospectus.pdf') %]" target="_blank">
+                                            [% l("Prospectus") %]
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="gray-line"></div>
+    <div class="center-text">
+        <img class="responsive timeline" src="[% request.url_for('images/pages/regulation/timeline.svg') %]">
+    </div>
+    <div>
+        <div class="background-gray center-text gr-padding-30 gr-12">
+            <h1 class="margin-top">[% l("Binary (Europe) Ltd") %]</h1>
+            <div class="gr-12">
+                <p>[% l("Binary (Europe) Ltd, Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta. Licensed and regulated (for gambling products only) by the Malta Gaming Authority in Malta (licence no MGA/CL2/118/2000) and for UK clients by the UK Gambling Commission - <a href=\"https://secure.gamblingcommission.gov.uk/gccustomweb/PublicRegister/PRSearch.aspx?ExternalAccountId=39495\" target=\"_blank\">view licence</a>.") %]</p>
```

Comment:
change "(for gambling products only)" to "(for gambling products only - Binary.com's Volatility Indices)"